plugins/services/src/js/containers/services/ServicesTable.js

```diff
@@ -391,6 +392,24 @@ class ServicesTable extends React.Component {
     );
   }
 
+  renderInstances(prop, service) {
+    const instancesCount = service.getInstancesCount();
+    const tasksRunning = service.getTaskCount();
+    const overview = tasksRunning === instancesCount
+      ? ` ${tasksRunning}`
+      : ` ${tasksRunning}/${instancesCount}`;
+
+    const content = typeof instancesCount !== "number"
```

Comment:
In theory `instancesCount` should always contain a number, so in case isn't we display a long dash.

Reply:
As `NaN` is a "number" you either add `isNaN` or use `isFinite` or `isInteger` to ensure a it's actually a valid instance count.